### DISCUSSION TITLE
Processing monetary data

### DISCUSSION THREAD

**Discussion by wickermann165** [appwrite/appwrite#3179] [Q&A] [answered]
Hello,
is there an easy way to process monetary data using aapwrite? The issue is that appwrite does not seem to support any decimal or similar data type.

Thanks

**Comment by maeddin** [comment, marked as answer]
So a general solution would be to take integers representing the smallest currency. Depending on what currency you want to use for it, of course. This is what I did in other projects where I didn't use AppWrite.

**Reply by wickermann165** [reply]
> So a general solution would be to take integers representing the smallest currency. Depending on what currency you want to use for it, of course. This is what I did in other projects where I didn't use AppWrite.

Thanks for sharing. Wanted to avoid this, but it seams necessary to use some kind of workaround for that if decimal or similar is not supported.

**Comment by stnguyen90** [comment]
What do you mean? There's float/double.

**Reply by maeddin** [reply]
With these data types you risk rounding errors.
